public/js/actions/sources.js

```diff
@@ -148,36 +156,6 @@ function closeTab(id) {
 }
 
 /**
- * Set the black boxed status of the given source.
```

Comment:
I removed the early support for blackbox in the source footer last week as it was not being used.